Box.V2/Managers/BoxSearchManager.cs

```diff
@@ -63,6 +63,7 @@ public BoxSearchManager(IBoxConfig config, IBoxService service, IBoxConverter co
                                                              
         {
 
+            keyword.ThrowIfNullOrWhiteSpace("keyword");
```

Comment:
@coolboy @marek-vysoky No, keyword CAN be null when you are doing a metadata-based query.  DO NOT change it from defaulting to null.